luckystrikehoreca/app/Http/Controllers/CateringController.php

```diff
@@ -20,57 +20,73 @@ public function getCateringItems()
         $cateringItems = CateringItem::all();
         // Fetch the unique_identifier from the store you get the second you enter the site
         $uniqueIdentifier = session('unique_identifier');
-        
         // Get the database object with this number and the date of today
         $userConnect = LoginUserConnect::where('unique_identifier', $uniqueIdentifier)
-        ->orderBy('date', 'desc')
-        ->first();
+            ->orderBy('date', 'desc')
+            ->first();
         // Group items by category
         $groupedItems = $cateringItems->groupBy('category');
         $reservation = $this->getCurrentUser($uniqueIdentifier);
-        
 
         if (!$reservation) {
             $user = new User();
             $user->name = 'Empty';
-        }else{
+        } else {
             $user = $reservation->user;
         }
 
-        return view('horeca')->with('cateringItems', $groupedItems)->with('laneId',$userConnect->unique_identifier)->with('user',$user);
+        return view('horeca')->with('cateringItems', $groupedItems)->with('laneId', $userConnect->unique_identifier)->with('user', $user);
     }
 
     private function getCurrentUser($id)
     {
         $lane = Lane::find($id);
         $reservation = Reservation::where('begin_time', '<=', Carbon::now())
-        ->where('end_time', '>=', Carbon::now())->where('lane_id','=',$lane->id)
-        ->get()->first();
+            ->where('end_time', '>=', Carbon::now())->where('lane_id', '=', $lane->id)
+            ->get()->first();
 
 
         return $reservation;
     }
 
     public function getOrderWithUser()
     {
+        // Set the time zone to 'Europe/Amsterdam'
+        date_default_timezone_set('Europe/Amsterdam');
+
         // Fetch the unique_identifier from the store you get the second you enter the site
         $uniqueIdentifier = session('unique_identifier');
-        
+
         // Get the database object with this number and the date of today
         $userConnect = LoginUserConnect::where('unique_identifier', $uniqueIdentifier)
-        ->orderBy('date', 'desc')
-        ->first();
+            ->orderBy('date', 'desc')
+            ->first();
 
         $reservation = $this->getCurrentUser($uniqueIdentifier);
 
         if (!$reservation) {
             $user = new User();
             $user->name = 'Empty';
-        }else{
+        } else {
             $user = $reservation->user;
         }
 
-        return view('order')->with('laneId',$userConnect->unique_identifier)->with('user',$user);
+        // Retrieve the orderData from the session
+        $orderData = session('orderData', []);
+
+        return view('order')
```

Comment:
return view('order', compact('laneId', 'user', 'orderData'));

much easier and less typing : )

Reply:
After this change it stopped working so I have not implemented this.